### DISCUSSION TITLE
When will Mojo be released 🔥 0.7

### DISCUSSION THREAD

**Discussion by Eprahim-taha** [modular/modular#1590] [Q&A] [answered]
Mojo 0.6 was released a month ago  and I wonder when Mojo 0.7 will be released.

**Comment by mojo-for-ai** [comment, marked as answer]
The last version of Mojo released by Modular was Mojo 0.6.1 dated 12/18/2023.  Since each release is a month apart from the one before it, the company is supposed to release another version of Mojo on 1/17/2024.
